### ISSUE TITLE
MSFT_AADGroup - adding members to a group fails

### ISSUE THREAD

**Issue by salbeck-sit** [microsoft/Microsoft365DSC#2203]
#### Details of the scenario you tried and the problem that is occurring
Create an AAD group in DSC with one user listed in Members. Applying the MOF fails

#### Verbose logs showing the problem
(name of server, group and member have been redacted)

VERBOSE: [MYSERVER]:                            [[AADGroup]e9ca50d6-5146-4f74-814f-8b2fbc75b540] Performing the
operation "Update-MgGroup_UpdateExpanded" on target "Call remote 'GroupsGroupUpdateGroup' operation".
VERBOSE: [MYSERVER]:                            [[AADGroup]e9ca50d6-5146-4f74-814f-8b2fbc75b540] Adding new member
{user@domain.com} to AAD Group {My-Group}
A parameter cannot be found that matches parameter name 'BodyParameter'.
    + CategoryInfo          : InvalidArgument: (:) [], CimException
    + FullyQualifiedErrorId : NamedParameterNotFound,New-MgGroupMemberByRef
    + PSComputerName        : localhost

VERBOSE: [MYSERVER]: LCM:  [ End    Set      ]  [[AADGroup]e9ca50d6-5146-4f74-814f-8b2fbc75b540]  in 0.9720
seconds.
The PowerShell DSC resource '[AADGroup]e9ca50d6-5146-4f74-814f-8b2fbc75b540' with SourceInfo 'path-to-DSC.ps1::185::9::AADGroup' threw one or more non-terminating errors while running the Set-T
argetResource functionality. These errors are logged to the ETW channel called Microsoft-Windows-DSC/Operational. Refer
 to this channel for more details.
    + CategoryInfo          : InvalidOperation: (:) [], CimException
    + FullyQualifiedErrorId : NonTerminatingErrorFromProvider
    + PSComputerName        : localhost

#### Suggested solution to the issue
Call New-MgGroupMember with -DirectoryObjectId instead of New-MgGroupMemberByRef

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
# insert configuration here
        AADGroup e9ca50d6-5146-4f74-814f-8b2fbc75b540
        {
            ApplicationId         = $ConfigurationData.NonNodeData.ApplicationId;
            CertificateThumbprint = $ConfigurationData.NonNodeData.CertificateThumbprint;
            DisplayName           = "My-Group";
            Ensure                = "Present";
            GroupTypes            = @();
            IsAssignableToRole    = $True;
            MailEnabled           = $False;
            MailNickname          = "1ff4bd28-8";
            Members               = @("user@domain.com");
            Owners                = @();
            SecurityEnabled       = $True;
            TenantId              = $ConfigurationData.NonNodeData.TenantId;
            Visibility            = "Private";
        }
```

#### The operating system the target node is running
OsName               : Microsoft Windows Server 2016 Datacenter
OsOperatingSystemSKU : DatacenterServerEdition
OsArchitecture       : 64-bit
WindowsBuildLabEx    : 14393.5246.amd64fre.rs1_release.220701-1744
OsLanguage           : en-US
OsMuiLanguages       : {en-US}

#### Version of the DSC module that was used ('dev' if using current dev branch)
'dev'

**Comment by NikCharlebois** [comment]
Looks like our documentation is not correct for the associated cmdlets (New-MgGroupOwnerByRef and New-MgGroupMemberByRef) https://docs.microsoft.com/en-us/powershell/module/microsoft.graph.groups/new-mggroupownerbyref?view=graph-powershell-1.0

The BodyParameter is not valid. Adding @peombwa as FYI

**Comment by salbeck-sit** [comment]
Adding or removing members are not part of unit-tests for the resource, so it might as well never have worked. That or it's a 'feature' of Graph in v. 1.11.0

**Comment by serial666** [comment]
@salbeck-sit That seems to be the culprit, with 1.10.0 this was working (albeit it doesn't check if the user exists or not to be added either as member or owner of the group) and with 1.11.0 the cmdlet doesn't have BodyParameter. Just checked and it was also removed from New-MgApplicationOwnerByRef which means my PR #2165 also stopped working.

**Comment by salbeck-sit** [comment]
The issue occurs using 1.,22.727.1 (ie Microsoft.Graph.* 1.11.0)

**Comment by serial666** [comment]
This is also fixed with 1.11.1, but IMHO it looks easier to the eye calling New-MgGroupOwner and Member directly than ByRef as explained on #2206, where I also included try/catch blocks around them and checks if users exist before trying to adding/removing them.

**Comment by salbeck-sit** [comment]
I can confirm that Graph v1.11.1 works with New-MgGroupOwner and New-MgGroupMember
